internal/network/manifests_generator.go

```diff
@@ -68,6 +69,81 @@ spec:
   osImageURL: ""
 `
 
+const snoDnsmasqConf = `
```

Comment:
I want it to be optional, dependent on ENV?

Reply:
optional == API to enable/disable it

Reply:
Need to think if there are cases where we don't want this internal DNS resolving for single node.

Reply:
yes, there is a case. The main customer of 4.8 (not writing the name here).
If it's prod there, maybe we would like to reduce the risk

Reply:
In general, OpenShift owns the resolution for api-int and can always point to whatever we want. the wildcard for apps is a different matter. I guess we could have a script that adds the record and SIGHUPs dnsmasq (with a new systemd service definition that contains reload) if there is no external DNS record for it.